notebooks/introduction_to_tensorflow/solutions/4_keras_functional_api.ipynb

```diff
@@ -185,7 +185,7 @@
     "\n",
```

Comment:
<p>Good catch! This makes sense! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>